salt/master.py

```diff
@@ -540,7 +540,7 @@ def start(self):
 
             if 'reactor' in self.opts and 'reactor' not in self.opts['engines']:
                 log.info('Enabling the reactor engine')
-                self.opts['engines']['reactor'] = {}
+                self.opts['engines'].append({'reactor': {}})
```

Comment:
Are you sure this change is right? It doesn't look like the master is starting correctly with it.